Monika After Story/game/script-topics.rpy

```diff
@@ -4825,6 +4825,26 @@ label monika_100k:
     return
 
 
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel='monika_kizuna',prompt="Virtual Youtuber?", category=['misc'],random=False,unlocked=False,pool=True, conditional="seen_event('greeting_hai_domo')"))
```

Comment:
In order for this to work, you also need an `action` arg, which is executed if the `conditional` is true. In your case, you'd want `action=EV_ACT_UNLOCK`